net/ngrep/Portfile

```diff
@@ -1,50 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem 1.0
+PortSystem              1.0
+PortGroup               github 1.0
 
-name                ngrep
-version             1.45
-revision            1
-categories          net
-maintainers         nomaintainer
+github.setup            jpr5 ngrep 1.48.3 v
+github.tarball_from     archive
+revision                0
+categories              net
+maintainers             nomaintainer
+license                 BSD/GLP/Artistic
```

Comment:
this isn't the correct way to specify the license. If all licenses apply it should be
`
BSD GLP Artistic
`

if there is a choice between either of them, then list them as
`{BSD GLP Artistic}`

Also, is `GLP` a typo and should it be `GPL` instead (if so specify a version number as well).